### PR TITLE
Feature/azaytsev/docs dlsteamer revision

### PR DESCRIPTION
**Pull request by andrew-zaytsev**
Revised DL Streamer documentation

### CODE REVIEW

**Review by tsavina — APPROVED** [review]

**Comment by mikhail-nikolskiy** [comment]
Hi,

Section name 'Media Processing' in documentation.md is confusing as it includes links to DL Streamer, OpenCV, OpenCL but all three are not really Media Processing libraries (Media Processing examples are oneVPL, FFMPEG, GStreamer, MediaFoundation). Can we split into two sections, Media Analytics and Additional Libraries, see below? Alternatively,
* if less wording expected, second sentence in Media Analytics section could be removed keeping first sentence
* may be rename Additional Libraries to 'OpenCV' and remove OpenCL as it's not distributed by OpenVINO

## Media Analytics
The OpenVINO™ toolkit also works with [Intel® DL Streamer](https://dlstreamer.github.io/), a streaming media analytics framework based on GStreamer*, for creating complex media analytics pipelines optimized for Intel hardware platforms. Go to Intel® DL Streamer [documentation](https://dlstreamer.github.io/) website to learn more and get information on how to download, install, and use.

## Additional Libraries
The OpenVINO™ toolkit comes with libraries and tools that add capability and flexibility to the toolkit. These include optimized versions of OpenCV and OpenCL.

File: docs/documentation.md [hunk 1]
```diff
@@ -103,4 +102,4 @@ The toolkit provides a [Performance Optimization Guide](optimization_guide/dldt_
 You can choose to use the [OpenVINO™ Deep Learning Workbench](@ref workbench_docs_Workbench_DG_Introduction), a web-based tool that guides you through the process of converting, measuring, optimizing, and deploying models. This tool also serves as a low-effort introduction to the toolkit and provides a variety of useful interactive charts for understanding performance.
 
 ## Media Processing
```
Comment (mikhail-nikolskiy) [inline docs/documentation.md:104]: Section name 'Media Processing' is confusing, see another comment with change proposal.

**Comment by andrew-zaytsev** [comment]
@mikhail-nikolskiy, thank you for your comments. Since neither of these products is now a component of the OpenVINO toolkit, I believe we don't need to introduce two sections just for one or two sentences. What about the following titles:
"Media Analytics and Computer Vision Tools", "Media Analytics and Other Tools"? For example,

## Media Analytics and Computer Vision Libraries

The OpenVINO™ toolkit also works with the following media processing frameworks and libraries:
* [Intel® DL Streamer](https://dlstreamer.github.io/) — A streaming media analytics framework based on GStreamer*, for creating complex media analytics pipelines optimized for Intel hardware platforms. Go to Intel® DL Streamer [documentation](https://dlstreamer.github.io/) website to learn more and get information on how to download, install, and use.
* [Intel® oneAPI Video Processing Library (oneVPL)](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/api-based-programming/intel-oneapi-video-processing-library-onevpl.html) — A programming interface for video decoding, encoding, and processing to build portable media pipelines on CPUs, GPUs, and other accelerators.

You can also add computer vision capabilities to your application using optimized versions of [OpenCV](https://opencv.org/) library.

**Comment by mikhail-nikolskiy** [comment]
Hi @andrew-zaytsev, thanks, your proposal looks good. One caveat as Vlatko noticed, there is legal requirement on first mention use full name Deep Learning Streamer, then ok to use DL Streamer. Here is slightly changed version according to this requirement:

## Media Analytics and Computer Vision Libraries

The OpenVINO™ toolkit also works with the following media processing frameworks and libraries:

* [Intel® Deep Learning Streamer (Intel® DL Streamer)](https://dlstreamer.github.io/) — A streaming media analytics framework based on GStreamer, for creating complex media analytics pipelines optimized for Intel hardware platforms. Go to Intel® DL Streamer [documentation](https://dlstreamer.github.io/) website to learn more.
* [Intel® oneAPI Video Processing Library (oneVPL)](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/api-based-programming/intel-oneapi-video-processing-library-onevpl.html) — A programming interface for video decoding, encoding, and processing to build portable media pipelines on CPUs, GPUs, and other accelerators.

You can also add computer vision capabilities to your application using optimized versions of [OpenCV](https://opencv.org/) and [OpenVX](https://khronos.org/openvx).

**Pull request merged**